packages/MainUI/app/not-found.tsx

```diff
@@ -0,0 +1,23 @@
+'use client';
+
+import Link from 'next/link';
+import Logo from '../public/etendo.svg?url';
+import Image from 'next/image';
+import { useTranslation } from '../hooks/useTranslation';
+import { Button } from '@mui/material';
+
+export default function NotFound() {
+  const { clientWidth, clientHeight } = window.document.body;
+  const { t } = useTranslation();
+
+  return (
+    <div className="center-all flex-column">
+      <Image src={Logo} width={clientWidth / 4} height={clientHeight / 4} unoptimized alt="Etendo" />
```

Comment:
Just a quick question for learning purposes @LuuchoRocha :)
Would it be more stable to use fixed rem units here instead 
of calculating clientWidth / 4? I’m thinking rem might provide more consistency across screen sizes

Reply:
@tomirodeghiero good question!

In this case, the width and height properties are not the actually values used for _rendering_ the image. This props are meant to calculate the aspect ratio of the image and not the actual sizes. **this is the behavior of the Next.js Image component and is not the default on other frameworks**

As [their docs](https://nextjs.org/docs/pages/api-reference/components/image#width) say:

This property is used to infer the correct aspect ratio of the image and avoid layout shift during loading. It does not determine the rendered size of the image, which is controlled by CSS, similar to the width attribute in the HTML <img> tag.

Reply:
Lol looks like I did not actually answer your original question. Given the implementation of the Next Image component, it's mandatory to provide **absolute** values and if you try to use **em**, **rem** or **%**, an error will be thrown.

My final solution was to add a CSS class that will apply different sizes (relative sizes, using %) based on screen orientation